### PR TITLE
[Part 1 - APIX Declaration Links] - Declaration Miner

### PR DESCRIPTION
**Pull request by josephaxisa**

### CODE REVIEW

File: packages/sdk-codegen-scripts/src/miner.ts [hunk 1]
```diff
@@ -74,13 +74,13 @@ export const filterAllFiles = (fileName: string) => fileName.trim().length > 0
...
  */
 export const filterCodeFiles = (fileName: string) => {
-  if (/(^|\/)node_modules|lib\//gi.test(fileName)) return false
+  if (/(^|\/)node_modules\//gi.test(fileName)) return false
   const ext = path.extname(fileName).toLocaleLowerCase()
   // eslint-disable-next-line @typescript-eslint/no-use-before-define
   return ext in fileMiners
 }
 
-const skipFolder = (name: string) => /node_modules|lib/gi.test(name)
```
Comment (jkaster) [inline packages/sdk-codegen-scripts/src/miner.ts:83]: Does this mean `lib` paths are now processed for example mining?

**Review by jkaster — COMMENTED** [review]
I have a question about how this may change example mining

File: packages/sdk-codegen-scripts/src/miner.ts (hunk 1 above)
Comment (josephaxisa) [reply packages/sdk-codegen-scripts/src/miner.ts:83]: My mistake. I just modified `skipFolder` to accept an exclude list

**Review by jkaster — APPROVED** [review]
LGTM

**Pull request merged**